typescript/ecs/ecs-service-with-task-placement/index.ts

```diff
@@ -26,18 +26,22 @@ const container = taskDefinition.addContainer('web', {
 container.addPortMappings({
   containerPort: 80,
   hostPort: 8080,
-  protocol: ecs.Protocol.Tcp
+  protocol: ecs.Protocol.TCP
 });
 
 // Create Service
 const service = new ecs.Ec2Service(stack, "Service", {
   cluster,
-  taskDefinition,
+  taskDefinition
 });
 
 // Specify binpack by memory and spread across availability zone as  placement strategies.
-// To place randomly, call: service.placeRandomly()
-service.placePackedBy(ecs.BinPackResource.Memory);
-service.placeSpreadAcross(ecs.BuiltInAttributes.AvailabilityZone);
+// To place randomly, add: ecs.PlacementStrategy.randomly() to the placement Strategy
+const placementStrategy =   [
```

Comment:
nit: extra spaces after =

Reply:
ad4ba178d26f18952367a8aed24dbaaa1b16330d addresses this issue.